Orange/regression/linear.py

```diff
@@ -14,6 +14,10 @@ class LinearRegressionLearner(SklLearner):
     __wraps__ = skl_linear_model.LinearRegression
     name = 'linreg'
 
+    def __init__(self, preprocessors=None):
+        super().__init__(preprocessors=preprocessors)
+        self.params = vars()
```

Comment:
This learner does not have any scikit-learn parameters so this line is unnecessary.